### PR TITLE
Added Preferred Provider Column to Weams

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description 
As a GIDS Tool user, I need the ability to upload a WEAMS.csv file containing a preferred provider indicator so that Comparison Tool users can identify preferred VET TEC providers.

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19179

## Testing done

1. Run `bundle exec rake db:migrate`
2. Ensure the stem indicator column is present in the database
3. Run `bundle exec rails s -p 4000`
4. Open the GI Bill Comparison Tool Dashboard
5. Export Weam.csv and ensure preferred_provider column is generated. 
6. Logged into the GIDS dashboard and exported a copy of the Weam CSV to ensure the preferred column was present. 
7. Logged into the GIDS dashboard and exported a copy of the Institutions CSV to ensure the preferred column was present. 

## Screenshots
Screen Shot 2019-07-10 at 9 21 05 AM

## Acceptance criteria
- [X] A WEAMS.CSV file containing a "Preferred Provider" column can be uploaded into GIDS.

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]
I think we should have a default false for preferred_provider.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Comment by ksantiagomorgan6** [comment]
@department-of-veterans-affairs/backend-review-group Could I get a code review on this at your earliest convenience. Thank you!" as a comment to the pull request.

File: spec/fixtures/weam.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
+10000008,MILTON POLICE DEPARTMENT,101 Federal Street,,,MILTON,DE,19968,USA,1740,N,N,APRVD,Educational Institution is Approved for All Chapters Except Chapter 31,No,Yes,No,No,No,,,N,101 Federal Street,,,MILTON,DE,19968,USA,1671,N,N,N,N,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          ,
```
Comment (jholton) [inline spec/fixtures/weam.csv]: Is the typical value for `Preferred Provider` in a WEAM upload going to be blank, like you left it here?
Or should it be `N` or `No`?

Comment (ksantiagomorgan6) [reply spec/fixtures/weam.csv]: It is going to be N, let me update that.

**Review by jholton — APPROVED** [review]
LGTM

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Comment by hughes-dustin** [comment]
@jholton Could we get another review on the new changes for this PR? We needed to add in changes so we could get preferred_provider data in the comparison tool search results.  Thanks!

**Review by jholton — APPROVED** [review]
LGTM

**Pull request merged**